source/mqtt_client_connection.c

```diff
@@ -87,6 +105,7 @@ static void s_mqtt_client_connection_finalize(napi_env env, void *finalize_data,
 
     /* Should have already been done, but just to be safe -- now that it's reentrant -- release the functions anyways */
     s_mqtt_client_connection_release_threadsafe_function(binding);
```

Comment:
If we are releasing the functions here, would we still need release in `s_on_connect_call`?